recipe/meta.yaml

```diff
@@ -27,7 +27,7 @@ requirements:
 test:
     commands:
         - iconv --help  # [not win]
-        - conda inspect linkages -n _test libiconv  # [linux]
+        - conda inspect linkages -p $CONDA_PREFIX libiconv  # [linux]
```

Comment:
I believe that `$PREFIX` is enough.
Can you also change `# [linux]` to `# [not win]` so this runs on `OS X` too?